client/src/components/Tags.tsx

```diff
@@ -18,9 +18,10 @@ export default function GenerateTags(props: { tags: string[], space?: number })
 }
 
 export function Tag(props: { tag: string }) {
+  const truncated = props.tag.length > 10 ? props.tag.slice(0, 7) + '...' : props.tag;
```

Comment:
This causes the tag to also be truncated in `ViewPostDialog` which looks weird. Only truncate on the previews